source/getstarted.html.erb

```diff
@@ -132,21 +132,19 @@ title: Get started
                 You must have a Payment Service Provider (PSP) to process the payments taken using GOV.UK&nbsp;Pay. However, there is no procurement work required to obtain a PSP.
                 This saves you time and money, and you’re not locked into a long-term contract.
               </p>
-              <p class="govuk-body">There are 2 PSP options available:</p>
+              <p class="govuk-body">The PSP you will use depends on the type of organisation you work for:</p>
```

Comment:
I'd change `you will` to `you'll`, in line with style guide advice on ctractions.

Reply:
```suggestion
              <p class="govuk-body">The PSP you'll use depends on the type of organisation you work for:</p>
```